docs/docs/guides/atom-effects.md

```diff
@@ -239,6 +241,80 @@ const currentUserIDState = atom({
 });
 ```
 
+### Asynchronous
```

Comment:
`localStorage` is only sync afaik.  For async examples let's move out of the "Local Storage" section into another subsection and not reference `localStorage`.

Reply:
That's very true. My original code was using `localForage` which is `async`, so it made sense in that case, but I converted that part to `localForage` since the other framework was related to my project and not recoil.

I'm uncertain how to write the documentation for this part then. Any suggestions?